doc.go

```diff
@@ -202,7 +202,13 @@ Operators and axis indicator
 	Scan (first axis)   ⍀    \%   +⍀B                       Running sum down B
 	Inner product       .    .    A+.×B        A +.* B      Matrix product of A and B
 	Outer product       ∘.   o.   A∘.×B        A o.* B      Outer product of A and B
-	                                                       (lower case o; may need preceding space)
+	                                                        (lower case o;
+	                                                        may need preceding space)
+	Each left                @f                A @f B       (A[1] f B), (A[2] f B), ...
+	                                                        as vector or matrix;
+	                                                        may need preceding space
```

Comment:
why would it need a preceding space?

Reply:
I thought it was an intentional feature of the lexer, but it was just a bug. Fixed.